x/docs/client/cli/tx.go

```diff
@@ -3,20 +3,33 @@ package cli
 import (
 	"bufio"
 
-	"github.com/cosmos/cosmos-sdk/client/flags"
-	uuid "github.com/satori/go.uuid"
-
-	ctypes "github.com/commercionetwork/commercionetwork/x/common/types"
-	"github.com/commercionetwork/commercionetwork/x/docs/internal/types"
 	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth"
 	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	uuid "github.com/satori/go.uuid"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 
-	"github.com/cosmos/cosmos-sdk/client/context"
-	"github.com/cosmos/cosmos-sdk/codec"
+	ctypes "github.com/commercionetwork/commercionetwork/x/common/types"
+	"github.com/commercionetwork/commercionetwork/x/docs/internal/types"
+)
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
+const (
+	FlagSign                    = "sign"
+	FlagSignStorageURI          = "sign-storage-uri"
+	FlagSignSignerInstance      = "sign-signer-instance"
+	FlagSignVcrID               = "sign-vcr-id"
+	FlagSignCertificateProfile  = "sign-certificate-profile"
+	FlagSignSdnDataFirstName    = "sign-sdn-data-first-name"
```

Comment:
All fields in `snd_data` are references to the x509 standard fields only. The values of the fields will be provided by another service after inserting the transaction in the chain